src/test/javascript/spec/helpers/mocks/service/mock-metis-service.service.ts

```diff
@@ -80,6 +80,10 @@ export class MockMetisService {
         return ['/courses', metisCourse.id!, 'discussion'];
     }
 
+    getLinkForExercise(exerciseId: string): string {
+        return '/courses/' + metisCourse.id + '/exercises/' + exerciseId;
```

Comment:
I'll just add suggestions for them all so you can directly commit from GitHub :D
```suggestion
        return `/courses/${metisCourse.id}/exercises/${exerciseId}`;
```